base/applications/games/spider/spider.cpp

```diff
@@ -365,7 +366,8 @@ LRESULT CALLBACK WndProc (HWND hwnd, UINT iMsg, WPARAM wParam, LPARAM lParam)
                     return 0;
 
                 case IDM_HELP_ABOUT:
-                    MessageBox(hwnd, MsgAbout, szAppName, MB_OK|MB_ICONINFORMATION);
+                    ShellAboutW(hwnd, szAppName, MsgAbout,
+                                LoadIconW(hInstance, MAKEINTRESOURCEW(IDI_SPIDER)));
```

Comment:
(Same comment as above)
```suggestion
                    ShellAbout(hwnd, szAppName, MsgAbout,
                               LoadIcon(hInstance, MAKEINTRESOURCE(IDI_SPIDER)));
```